src/acquisition/covid_hosp/common/utils.py

```diff
@@ -168,15 +163,13 @@ def merge_by_key_cols(dfs, key_cols, logger=False):
     return result.reset_index(level=key_cols)
 
   @staticmethod
-  def update_dataset(database, network, newer_than=None, older_than=None):
+  def update_dataset(database, newer_than=None, older_than=None):
```

Comment:
Done and I moved two other methods (`issues_to_fetch`, `merge_by_key_cols`) from `Utils` into `Database` as they were only used there.

Reply:
nice, now those two other methods dont need the logger argument or the `if logger` checks!  `issues_to_fetch()` almost feels like it might better belong in the `Network` class, but its fine to stay where it is.

theres not much left in this file now...  should we move the remaining methods into `/src/common/covid_hosp/covid_hosp_schema_io.py`?  theyre only used there.

Reply:
Hm, not sure it's possible to cleanly do that. In `CovidHospSomething` (name pending :), `PYTHON_TYPE_MAPPING` refers to `Utils.parse_bool`, `Utils.int_from_date` and `Utils.limited_geocode`. But if those methods were to be moved to `CHS`, `PYTHON_TYPE_MAPPING` wouldn't be able to reference them as it's defined in the class body.

Reply:
theres got to be a way to do that thats cleaner than what exists currently.  its awkward to import things from deep in `/src/acquisition/covid_hosp/common/` into `src/common/covid_hosp/` when theyre only used in the latter.